frontend/src/lobby/lobby.css

```diff
@@ -0,0 +1,105 @@
+.lobby-container {
+    display: flex;
+    flex-direction: row;
+    width: 80%;
+    height: 300px;
+    border-radius: 15px;
+    background-color: #f5f5f5; 
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    margin: 20px auto;
+    overflow: hidden;
+  }
+  
+  .left-column {
+    flex: 1;
+    background-color: #e0e0e0; 
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 20px;
+  }
+  
+  .right-column {
+    flex: 2;
+    margin: 0px 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    font-family: Arial, sans-serif;
+  }
+  
+  .text-box {
+    width: 75%;
+    padding: 10px;
+    font-size: 16px;
+    border-radius: 5px;
+    border: 1px solid #ccc;
+  }
+  
+  .heading {
+    color: #282c34; 
+    font-size: 24px;
+    margin-bottom: 15px;
+  }
+  
+  .rooms-table-container {
+    width: 100%;
+    height: 150px; /* Adjusted height for table scroll */
+    overflow-y: auto; /* Enable scrolling */
+    margin-top: 10px;
+  }
+  
+  .rooms-table {
+    width: 100%;
+    border-collapse: collapse;
+  }
+  
+  .rooms-table th, .rooms-table td {
+    border: 1px solid #ddd;
+    padding: 8px;
+    text-align: left;
+  }
+  
+  .rooms-table th {
+    background-color: #e0e0e0; 
+    color: #282c34; 
+  }
+  
+  .rooms-table td {
+    background-color: #f5f5f5; 
+    color: #282c34; 
+  }
+  
+  button {
```

Comment:
this change applies to all the buttons in the application, causing changes to the current login/signup buttons. Maybe identify the buttons with a lobby classname so it only changes the ones in lobby.
![image](https://github.com/user-attachments/assets/ae2a1e44-854b-413c-af70-2f93add8bd69)

Reply:
Changed it to only affect the button in the Lobby. The login buttons remain with their style.